### PR TITLE
CYF-Glasgoow | Iakub Dubachev | Book Library | Sprint 2

### PR DESCRIPTION
**Pull request by IakubDubachev**
Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/contributing/)
- [ ] My changes meet the [requirements](./README.md) of this task

Changelist

- Fixed book list not displaying on page load.

- Corrected author field assignment and added input validation in submit().

- Fixed delete button functionality (clicks → click).

- Corrected read status display logic (Yes/No).

- Improved table rendering to handle dynamic updates.

- Added clearForm() to reset input fields after adding a book.

Questions

- Are there any additional edge cases I should test?

- Any recommendations for improving DOM updates or event handling?

### CODE REVIEW

File: debugging/book-library/script.js [hunk 1]
```diff
@@ -53,51 +63,40 @@ function Book(title, author, pages, check) {
...
-    delBut.innerHTML = "Delete";
-    delBut.addEventListener("clicks", function () {
-      alert(`You've deleted title: ${myLibrary[i].title}`);
-      myLibrary.splice(i, 1);
+    delButton.className = "btn btn-warning";
+    delButton.innerText = "Delete";
+    deleteCell.appendChild(delButton);
+
+    delButton.addEventListener("click", function () {
+      alert(`You've deleted title: ${book.title}`);
```
Comment (shieldo) [inline debugging/book-library/script.js:97]: A really small thing is that I would show this alert after you have performed the re-render - this would probably be less surprising for a user!

This is good work, though! I like the way that you have introduced ternary expressions, for example.

**Review by shieldo — APPROVED** [review]
